src/main/frontend/fs/sync.cljs

```diff
@@ -2980,11 +2976,44 @@
 
     (reset! current-sm-graph-uuid nil)))
 
+(defn <sync-local->remote []
```

Comment:
how about rename to `<sync-local->remote-now`, because there's a fn called `<sync-local->remote!`

Reply:
done